src/ol/mapbrowsereventhandler.js

```diff
@@ -281,8 +279,8 @@ ol.MapBrowserEventHandler.prototype.relayEvent_ = function(pointerEvent) {
  * @private
  */
 ol.MapBrowserEventHandler.prototype.isMoving_ = function(pointerEvent) {
-  return pointerEvent.clientX != this.down_.clientX ||
-      pointerEvent.clientY != this.down_.clientY;
+  return Math.abs(pointerEvent.clientX - this.down_.clientX) > 1 ||
+      Math.abs(pointerEvent.clientY != this.down_.clientY) > 1;
```

Comment:
Typo: the second condition should be `pointerEvent.clientY - this.down_.clientY`.

One question: Any chance the threshold `1` may be too large? The real offset that are causing the bug seems well below `1`. May be `0.5` or even `0.9` is a better alternative?